openedx/core/djangoapps/user_authn/views/logout.py

```diff
@@ -17,10 +17,11 @@
 from openedx.core.djangoapps.user_authn.utils import is_safe_login_or_logout_redirect
 
 from third_party_auth.saml import SAMLAuthBackend
-
+from third_party_auth import pipeline, provider
+import third_party_auth
 SAML_BACKEND = 'tpa-saml'  # Keyclock Saml Backend
 LEARNING_PORTAL = 'http://localhost:8734'  # Learning portal local domain name.
-LMS_HOST = 'http://localhost:18000/'
+KEY_CLOCK_LOGOUT_URL ='{}/protocol/openid-connect/logout?redirect_uri={}'
```

Comment:
you should not hardcode the url path. It should be pulled from config.

Reply:
Keyclock config not providing us any logout URL for SAML or OpenId.